AutomatedTesting/Gem/PythonTests/prefab/TestSuite_Main.py

```diff
@@ -29,3 +30,14 @@ def _run_prefab_test(self, request, workspace, editor, test_module):
     def test_PrefabLevel_OpensLevelWithEntities(self, request, workspace, editor, launcher_platform):
         from . import PrefabLevel_OpensLevelWithEntities as test_module
         self._run_prefab_test(request, workspace, editor, test_module)
+
+    @pytest.mark.parametrize("level", ["tmp_level"])
+    def test_PrefabLevel_BasicWorkflow(self, request, workspace, editor, launcher_platform, level):
+        def teardown():
+            file_system.delete([os.path.join(workspace.paths.project(), "Levels", level)], True, True)
+        request.addfinalizer(teardown)
+        file_system.delete([os.path.join(workspace.paths.project(), "Levels", level)], True, True)
```

Comment:
Creating temporary levels makes the test to be non-parallelizable along with other problems.
Instead of creating a new level, just open an existing Base or Empty level and just make sure to not save it once finished

Reply:
Will add 'Base' level in Prefab/ just like what's done in physics/ .

Reply:
Fixed.